Documentation/AudioFileFormats.md

```diff
@@ -0,0 +1,93 @@
+# Audio Formats Produced by Libation
+
+Libation will download audio in a number of different audio formats, depending on the settings you choose within Libation and the per-title availability of audio formats from Audible. The Libation settings which affect the format downloaded by Libation are shown in the Settings menu screenshot below.
+
+Notes:
+- Audiobook file extensions are either `.m4b` or `.mp3`. Libation uses the `.m4b` file extension for all non-MP3 files, regardless of the audio codec contained therein. Some media players don't recognize the `.m4b` file extension and may require the extension be changed to `.m4a` or `.mp4`.
+- Most (but not all) podcasts are delivered by Audible as native MP3 files. None of the following audio formats and settings discussions pertain to those podcasts because MP3s have no DRM, and those episodes are copied directly to their output folders.
+
+<img width="822" height="173" alt="image" src="https://github.com/user-attachments/assets/da9adf57-a8da-4962-a6e2-94d86883ba87" />
+
+## Settings Summary
+### Audio quality to request from Audible
+Audiobooks can be requested from Audible as "Normal" quality or "High" quality, matching the settings in the Audible mobile apps. This setting affects the audio bitrate and, sometimes, the number of audio channels. This setting has no effect on the _audio codec_.
+
+### Use Widevine DRM
+This setting will request Audio files protected by Google's Widevine Digital Rights Managements scheme. The practical effects of this option are:
+
+1. When disabled, all audiobooks will be downloaded in the [AAC-LC codec](#aac-lc).
+2. When enabled, (nearly) all non-spatial audiobooks will be downloaded in the [xHE-AAC codec](#xhe-aac). 
+3. This option must be enabled to enable "Request Spatial Audio"
+
+### Request Spatial Audio
+Enable this setting to request audiobooks in a "spatial" ([Dolby Atmos](#dolby-atmos)) audio format. If an audiobook is not available in a spatial format, it will instead be downloaded in the [xHE-AAC codec](#xhe-aac).
+
+### Spatial audio codec
+Choose whether spatial audiobooks are downloaded in the [E-AC-3](#e-ac-3) or [AC-4](#ac-4) format.
+
+### Download my books in the original audio format (Lossless)
+If selected, Audiobooks will be downloaded and saved in the format delivered by audible (which depends on the settings explained above). Libation will not change the audio.
+
+### Download my books as .MP3 files (transcode if necessary).
+If selected, Libation will decode [AAC-LC](#aac-lc), [xHE-AAC](#xhe-aac), and [E-AC-3](#e-ac-3) audiobooks and re-encode them as MP3s using the MP3 encoder settings ([read about LAME MP3 encoder settings](https://lame.sourceforge.io/lame_ui_example.php)). Note that Libation cannot convert [AC-4](#ac-4) audio to MP3.
```

Comment:
Will Libation figure out which audio to request if we've checked all of the fancy settings and then click convert to MP3?